src/services/AxiosPrivateClient.tsx

```
@@ -36,50 +39,76 @@ const useSetupAxiosInstance = () => {
+          'x-signature': signature,
+          'x-signing-message': toHex(String(signingMessage)),
+        }
+
+        await axiosInstance.post('/auth/login', { client }, { headers, withCredentials: true })
+        resolve()
+      } catch (error) {
+        reject(error)
+      } finally {
+        signingPromise = null
+      }
+    })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)